indico/modules/events/management/templates/contribution_person_list.html

```diff
@@ -60,6 +67,18 @@
                                 -
                             {% endif %}
                         </td>
+                        {% if include_submitters %}
+                            {% set is_submitter = person_roles.submitter %}
+                            <td class="i-table submitter-column">
+                                {%- if is_submitter %}
+                                    {% set submitter_title = _('This person is a submitter') %}
```

Comment:
"This person submitted an abstract"? "is a submitter" sounds more like people with some kind of submission privs